### DISCUSSION TITLE
Loki query dashboard supports adding same label which was already chosen

### DISCUSSION THREAD

**Discussion by SanthoshS20** [grafana/grafana#53604] [Observability]
**What happened:**
The label dropdown list shows the already chosen label in the query field. We can able to choose same label with same value multiple times which is not required.

**What you expected to happen:**
We should remove the labels from the dropdown list which was already chosen.

**How to reproduce it (as minimally and precisely as possible):**
**Step 1:** Go the the Loki dashboard.
**Step 2:** Switch to the builder mode.
**Step 3:** In the labels, click the + (PLUS) button to add a new label.
**Step 4:** Choose a label.
**Step 5:** Add another label by clicking the + (PLUS) button.
**Step 6:** Choose the same label which is already chosen.

**Anything else we need to know?:**
![183443387-8dc58307-0c87-40ce-b981-c7242d7b4a00](https://user-images.githubusercontent.com/37366893/184168568-afca6182-3bbc-48b2-9f31-019ec4be7518.png)

**Environment:**

Grafana version: 9.0.5
User OS & Browser: Windows 10 & Chrome Browser

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
